test/fakeintake/aggregator/logAggregator_test.go

```diff
@@ -19,7 +19,13 @@ var logData []byte
 
 func TestLogAggregator(t *testing.T) {
 	t.Run("parseLogPayload should return empty log array on empty data", func(t *testing.T) {
-		checks, err := ParseLogPayload(api.Payload{Data: []byte(""), Encoding: encodingGzip})
+		checks, err := ParseLogPayload(api.Payload{Data: []byte(""), Encoding: encodingEmpty})
```

Comment:
Noticed that the encoding was set to `gzip` when it should be empty, correct me if I'm wrong